examples/gpt3/config.yaml

```diff
@@ -0,0 +1,21 @@
+data_dir: data
+environment_variables: {}
+examples_filename: examples.yaml
+input_type: Any
+model_class_filename: model.py
+model_class_name: GPT3Model
+model_framework: custom
+model_metadata: {}
+model_module_dir: model
+model_name: null
+model_type: custom
+python_version: py39
+requirements:
+  - openai==0.6.3
```

Comment:
there's a [much newer](https://github.com/openai/openai-python) version of the library now - seems like a good idea to upgrade